enrollment-server/src/main/java/com/wultra/app/enrollmentserver/controller/api/InboxController.java

```diff
@@ -68,7 +68,7 @@ public InboxController(PushServerClient pushClient) {
             PowerAuthSignatureTypes.POSSESSION_KNOWLEDGE,
             PowerAuthSignatureTypes.POSSESSION_KNOWLEDGE_BIOMETRY
     })
-    public ObjectResponse<GetInboxCountResponse> countUnreadMessages(@RequestBody ObjectRequest<GetInboxCountRequest> objectRequest, @Parameter(hidden = true) PowerAuthApiAuthentication apiAuthentication) throws InboxException {
+    public ObjectResponse<GetInboxCountResponse> countUnreadMessages(@Parameter(hidden = true) PowerAuthApiAuthentication apiAuthentication) throws InboxException {
```

Comment:
We should also delete the unused class `GetInboxCountRequest`, but I don't know whether it's correct to declare method as "POST" and then use no post payload.

Reply:
@hvge The method definition says that the incoming request body is not processed. We do not need to process the request body since there is no usable payload. So yes - this approach is correct. Good point about deleting unused class, though!